### ISSUE TITLE
Stuck by BLAST in LTR finding 

### ISSUE THREAD

**Issue by Yutang-ETH** [oushujun/EDTA#453]
Hi Shujun, 

I hope you are well. 

I am running EDTA v2.2.1 for a haplotype-resolved genome assembly of a diploid forage grass genome with an assembly size of 4.4 Gb (2.2 Gb per haplome). However, I am always stuck in LTR finding by some dead BLAST jobs (please see attached screenshot). I am running EDTA with 60 CPUs and 900 Gb RAM, and our disk has enough space (5 TB available). Besides, I have tested EDTA on our machine with the test data, everything went smoothly. Any ideas why EDTA is stuck by BLAST in LTR finding, and are there any solutions for this? 

Thank you very much.

Best wishes,
Yutang
![Picture2](https://github.com/oushujun/EDTA/assets/84848653/5a39c121-0321-4c49-a31c-610d67bb5ab1)

**Comment by oushujun** [comment]
Blast may get stuck for some reason. You may kill the process and restart. If the issue persist, you can extract that line of code and run it manually or submit it here for me to test.

Thanks!
Shujun

**Comment by Yutang-ETH** [comment]
Thank you very much Shujun for your reply. I see, I will report the sequence to you if I get stuck by the same blast job again.  

While I was waiting for your reply, I tried to run each haplotype assembly separately, and this time I used --sensitive 0 (I was using --sensitive 1 for my last run with the diploid assembly. Sorry for not mentioning this at the beginning).  So far, EDTA runs well without getting stuck at the LTR finding step. I am not sure if I am lucky for this run or it is that --sensitive 1 increases the chance of getting stuck by blast. 

Anyway, I close the issue for the moment, and I thank you very much for your communication again. 

Best wishes,
Yutang
